### PR TITLE
[INT-1312][node-onfleet] Delivery Manifest endpoint

### PR DESCRIPTION
**Pull request by lisandrobigi-onfleet**
**Describe the solution**
New Onfleet Delivery Manifest API endpoint added as `getDeliveryManifest` method. In order to test the mentioned endpoints, the test file was updated using a dummy parameter.
https://docs.onfleet.com/reference/delivery-manifest

---

**Added**

- Added getDeliveryManifest endpoint
- Support to include Google Maps Directions API key, in this specific format Google <google_direction_api_key> in your HTTP header as X-API-KEY
- Support for optional query parameters `startDate` and `endDate`.
- The test file was updated
- Readme files were updated to support Workers POST method.

**Changed**

**Deprecated**

**Removed**

**Fixed**

**Security**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -256,4 +260,11 @@ describe('HTTP Request testing', () => {
         assert.equal(res.tasks[0].id, '3VtEMGudjwjjM60j7deSI987');
       });
   });
+  it('Get compliance information from tasks assigned to Onfleet drivers', () => {
```
Comment (danmenza) [inline test/test.js:263]: I would also try including one other unit test in here to confirm that the `turnByTurn` object isn't empty when a google API key is present. Not required though, would just be a good test to include if it's not a ton of work

**Review by danmenza — CHANGES_REQUESTED** [review]
Added some comments, let me know if we need to discuss those optional query parameters in greater detail. Overall, pretty close good work.

File: test/test.js (hunk 1 above)
Comment (lisandrobigi-onfleet) [reply test/test.js:263]: Done

File: lib/Methods.js [hunk 2]
```diff
@@ -110,6 +110,17 @@ const Methods = (key, api, ...args) => {
       }
     }
   }
+  // Reference https://docs.onfleet.com/reference/delivery-manifest
+  if (path.includes("integrations")) {
```
[the remark this answers is not in the document]
Comment (lisandrobigi-onfleet) [reply lib/Methods.js]: Changed for a better and more understandable check

**Review by danmenza — APPROVED** [review]
Looks good, nice refactor. Approved ✅
